linux/tools.Dockerfile

```diff
@@ -86,9 +86,7 @@ RUN ltarget=$(readlink /usr/local/linkerd/bin/linkerd) && \
     if [ ! -f $ltarget ] ; then rm /usr/local/linkerd/bin/linkerd ; ln -s /usr/local/linkerd/bin/linkerd-stable* /usr/local/linkerd/bin/linkerd ; fi
 
 # Temp: fix ansible modules. Proper fix is to update base layer to use regular python for Ansible.
-RUN wget -nv -q https://raw.githubusercontent.com/ansible-collections/azure/dev/requirements-azure.txt \
-    && /opt/ansible/bin/python -m pip install -r requirements-azure.txt \
-    && rm requirements-azure.txt
+RUN /opt/ansible/bin/python -m pip install -r ~/.ansible/collections/ansible_collections/azure/azcollection/requirements-azure.txt
```

Comment:
Is there a reason ansible python interpreter is being used instead of the python already available in cloud shell base image? If there is a specific reason, we could keep it as it is to avoid base image dependency for things in tools image.

Reply:
Tested on local machine,  base image needs to be updated otherwise the Ansible will not find the required dependencies. Here I just update the dependency `requirements-azure.txt` to the specific azcollection version, not the latest `requirements-azure.txt`.

Reply:
Q1: Can you please tell us what changes you are planning to make to base image?
Q2: Can't we just use python3 that is already available from base image over here? why use a specific python interpreter /opt/ansible/bin/python? Does pip install only work with this particular version of python?
Q3: If you are not using `wget` to get the requirements, where are you downloading the requirements again?
Q4: Does ansible installation in base image also install python? Then we can ignore question 2.

Reply:
Q1: I will  not change anything in the base image 
Q2: This is an isolate python environment for Ansible azcollection only, all the dependencies will only aviable in this env and will not effect or be effect by other tools python dependnecies.
Q3: `wget` will install the latest azcollection requirements which may not suit for the release version. Changed the requirements to use the installed azcollection `requirement-azure.txt`
Q4: No, base image create a new python virtual enevironment for Ansible azcollection.

Reply:
LGTM!